packages/workbox-cli/src/lib/questions/ask-root-of-web-app.ts

```diff
@@ -17,8 +17,9 @@ import {constants} from '../constants';
 
 const ROOT_PROMPT = 'Please enter the path to the root of your web app:';
 
-// The key used for the question/answer.
-const name = 'globDirectory';
+// The keys used for the questions/answers.
+const question_rootDirectory = 'globDirectory';
```

Comment:
Can you switch this and the other variable to `questionRootDirectory` instead of `question_rootDirectory`?

That's more in keeping with our variable naming pattern.

Reply:
Hey @jeffposnick, thank you for going through the PR and reporting back.
I'll be sure to address the changes asap 🙂 .